tests/beamlines/unit_tests/test_i24.py

```diff
@@ -4,7 +4,10 @@
 
 
 @pytest.mark.parametrize("module_and_devices_for_beamline", ["i24"], indirect=True)
-def test_device_creation(RE, module_and_devices_for_beamline):
+def test_device_creation(
+    RE,
+    module_and_devices_for_beamline,
+):
```

Comment:
Can this unrelated formatting change (if it doesn't break the CI) be reverted?

Reply:
done

Reply:
Not quite, a space snuck in

Reply:
done